packages/icons/src/blueprint-icons.scss

```diff
@@ -0,0 +1,5 @@
+// Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the terms of the LICENSE file distributed with this project.
+
+@import "font-imports";
```

Comment:
don't need this here, it's just a mixin.

Reply:
...what imports it then? I moved this over from an import in core. How's it used?

Reply:
oh nvm I thought `font-face`. this is good 👍